src/tests/getkeys.c

```diff
@@ -41,7 +41,7 @@ int main(int argc, char *argv[])
 		goto out;
```

Comment:
Similar comment: until LXC 3.0 we should still run tests for the old configuration keys. So this file should only test for both `lxc.network.{[i]}.*` and `lxc.net.[i].*` keys.